### ISSUE TITLE
Parcel support

### ISSUE THREAD

**Issue by mbaumgartl** [tus/tus-js-client#193]
Hi!

I try to build an uploader using tus-js-client, but I struggle at the very beginning. Parcel bundles the wrong code and this results in a very big bundle file:

```
public/dist/bundle.js                                                                   
├── node_modules/bn.js/lib/bn.js                                                        
├── node_modules/diffie-hellman/node_modules/bn.js/lib/bn.js                            
├── node_modules/elliptic/lib/elliptic/precomputed/secp256k1.js                         
├── node_modules/buffer/index.js                                                        
├── node_modules/tus-js-client/lib.esm/upload.js                                        
├── node_modules/elliptic/lib/elliptic/curve/short.js                                   
├── node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js         
├── node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_readable.js   
├── node_modules/readable-stream/lib/_stream_readable.js                                
├── node_modules/lodash._baseiteratee/index.js                                          
├── node_modules/asn1.js/lib/asn1/base/node.js                                          
├── node_modules/util/util.js                                                           
├── node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_writable.js   
├── node_modules/hash-base/node_modules/readable-stream/lib/_stream_writable.js         
├── node_modules/url/url.js                                                             
├── node_modules/readable-stream/lib/_stream_writable.js                                
├── node_modules/diffie-hellman/lib/primes.json                                         
├── node_modules/assert/node_modules/util/util.js                                       
├── node_modules/elliptic/lib/elliptic/curve/edwards.js                                 
├── node_modules/events/events.js                                                       
└── + 230 more assets                                                                
```

This is my test code:
```js
import { Upload } from 'tus-js-client';

document.addEventListener('DOMContentLoaded', () => {
    console.log(Upload);
}, false);
```

If I change the import to
```js
import { Upload } from 'tus-js-client/lib/browser/index';
```
the correct files are bundled:
```
public/dist/bundle.js                                        
├── node_modules/tus-js-client/lib/upload.js                 
├── node_modules/js-base64/base64.js                         
├── node_modules/url-parse/index.js                          
├── node_modules/tus-js-client/lib/browser/fileReader.js     
├── node_modules/tus-js-client/lib/error.js                  
├── node_modules/tus-js-client/lib/browser/index.js          
├── node_modules/tus-js-client/lib/browser/httpStack.js      
├── node_modules/tus-js-client/lib/browser/urlStorage.js     
├── node_modules/querystringify/index.js                     
├── node_modules/tus-js-client/lib/browser/fingerprint.js    
├── node_modules/tus-js-client/lib/noopUrlStorage.js         
├── node_modules/tus-js-client/lib/browser/uriToBlob.js      
├── node_modules/tus-js-client/lib/browser/readAsByteArray.js
├── node_modules/requires-port/index.js                      
├── node_modules/tus-js-client/lib/browser/isReactNative.js  
├── node_modules/tus-js-client/lib/uuid.js                   
├── node_modules/tus-js-client/lib/browser/isCordova.js      
├── src/main.js                                              
└── node_modules/tus-js-client/lib/logger.js                 
```
My package.json is quiet simple too:
```json
{
  "name": "upload",
  "version": "1.0.0",
  "description": "",
  "scripts": {
    "dev": "parcel watch --no-hmr --no-autoinstall --out-dir public/dist/ --public-url /dist/ src/main.js",
    "build": "parcel build --out-dir public/dist/ --out-file bundle.js src/main.js"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {},
  "devDependencies": {
    "parcel-bundler": "^1.12.4",
    "tus-js-client": "^2.0.1"
  },
  "browserslist": [
    "last 2 Chrome versions",
    "Firefox ESR"
  ]
}
```
Misconfiguration in my project, bug in Parcel or problem in tus-js-client?

Thanks in advance!

Regards,
Marco

**Comment by Acconut** [comment]
Thanks for reporting. That was a bug on our side and should be fixed in the 2.0.2 release which will be available in a few minutes. Please try and report back if it does not work.

**Comment by mbaumgartl** [comment]
Thanks for the fix und the quick response. Works for me! :grin:
